tests/PHPStan/Analyser/NodeScopeResolverTest.php

```diff
@@ -309,6 +309,9 @@ public function dataFileAsserts(): iterable
 		require_once __DIR__ . '/../Rules/Generics/data/bug-3769.php';
 		yield from $this->gatherAssertTypes(__DIR__ . '/../Rules/Generics/data/bug-3769.php');
 
+		require_once __DIR__ . '/../Rules/Generics/data/bug-6301.php';
```

Comment:
No need to require it. If it failed locally for you, you forgot to run `composer dump` before running the tests :)